test/sharness/t0090-get.sh

```diff
@@ -154,30 +154,9 @@ test_get_cmd() {
   '
 }
 
-test_get_fail() {
-  test_expect_success "create an object that has unresolvable links" '
-    cat <<-\EOF >bad_object &&
-{ "Links": [ { "Name": "foo", "Hash": "QmZzaC6ydNXiR65W8VjGA73ET9MZ6VFAqUT1ngYMXcpihn", "Size": 1897 }, { "Name": "bar", "Hash": "Qmd4mG6pDFDmDTn6p3hX1srP8qTbkyXKj5yjpEsiHDX3u8", "Size": 56 }, { "Name": "baz", "Hash": "QmUTjwRnG28dSrFFVTYgbr6LiDLsBmRr2SaUSTGheK2YqG", "Size": 24266 } ], "Data": "\b\u0001" }
-EOF
-    cat bad_object | ipfs object put > put_out
-  '
-
-  test_expect_success "output looks good" '
-    echo "added QmaGidyrnX8FMbWJoxp8HVwZ1uRKwCyxBJzABnR1S2FVUr" > put_exp &&
-    test_cmp put_exp put_out
-  '
-
-  test_expect_success "ipfs get fails" '
-    test_expect_code 1 ipfs get QmaGidyrnX8FMbWJoxp8HVwZ1uRKwCyxBJzABnR1S2FVUr
-  '
-}
```

Comment:
We can't add an invalid object with `dag put`.

Reply:
We can with `block put`. Mind keeping this test, but refactoring it to use that?

Reply:
>We can't add an invalid object with dag put

The object is valid right? IIUC the issue is that the JSON encoding format isn't the same since `dag put` wants the data to be encoded with a given codec (e.g. raw, dag-json, dag-cbor) and there isn't a codec representing that specific JSON-dagpb representation.

Whether you use `block put` or `dag put` you'll need to change what the input looks like anyway (even though the resulting CID should be the same)

Reply:
@aschmahmann you're very right. I will bring it back.